### PR TITLE
main body and assets

### PR DESCRIPTION
**Pull request by Exequielt**

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -1,12 +1,127 @@
...
+      <li class="move-left">NORTH AMERICA</li>
+      <li class="button1">MORE INFO</li>
+    </ul>
+
+      <div class="border"></div>
+
+    
+      
+      <ul class="move-right">
+        <li >SOUTH AMERICA</li>
```
Comment (nathan-hyan) [inline Index.html]: remove spaces inside tag `<li>`

File: Index.html [hunk 2]
```diff
@@ -1,12 +1,127 @@
...
+      <ul class="button1">
+      <li>MORE INFO</li>
+      </ul>
+    </ul>
+
+  </div>
+  
+  <div class="border"></div>
+
+    <footer>
```
Comment (nathan-hyan) [inline Index.html]: This page doesn't have a footer, so this can be replaced with a div

File: Styles/index.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
+@import "variables";
+@import "index-2";
+@import "index-3";
+@import "index-4";
+@import "index-5";
```
Comment (nathan-hyan) [inline Styles/index.scss]: use descriptive names, not numbers

File: Styles/_index-3.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  border-radius: 3px;
+  padding: 15px 32px;
+  text-align: center;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.button3 {
+  background-color: $--yellow;
+  border: transparent;
```
Comment (nathan-hyan) [inline Styles/_index-3.scss]: Border is transparent by default, its redundant to re-declare it.

**Review by nathan-hyan — COMMENTED** [review]
comments

File: Index.html [hunk 5]
```diff
@@ -1,12 +1,127 @@
...
+          <li class="title-2">YOUR JOURNEY BEGINS HERE</li>
+          <li class="button1">START NOW</li>
+      </ul>
+      <ul>
+         <li class="logo-size"><img src="./Assets/world.webp" alt="Mouse-Track" /></li>
+         <li class="title-2b">millions of competitors around the world are already warming-up for the
+          biggest event of their lives.</li>
+      </ul>  
+    </ul>
+  </div>
```
Comment (FernandoRApud) [inline Index.html]: Tab spacing not consistent (View the div closing tag)

File: Index.html [hunk 6]
```diff
@@ -1,12 +1,127 @@
...
+          <li>ABOUT</li>
+        </ul>
+
+        <ul class="portfolio">
+          <li>VOLVER AL PORTFOLIO</li>
+        </ul>
+      </div>
+    </header>
+
+    <div class="border">
```
Comment (FernandoRApud) [inline Index.html]: Change this div for `<hr>` tag. Style it, if necessary.

File: Index.html [hunk 7]
```diff
@@ -1,12 +1,127 @@
...
     
-</body>
-</html>
\ No newline at end of file
+      <li class="button2">LEARN MORE</li>
+    
+    </ul>
+  </div>
+
+  <div class="border"></div>
```
Comment (FernandoRApud) [inline Index.html]: Same as 29 line

File: Index.html [hunk 8]
```diff
@@ -1,12 +1,127 @@
...
+    <ul class="align">
+      <li class="list-4 button3">CHOOSE YOUR DESTINATION</li>
+    </ul>
+
+    <ul>
+      <li class="move-left">NORTH AMERICA</li>
+      <li class="button1">MORE INFO</li>
+    </ul>
+
+      <div class="border"></div>
```
Comment (FernandoRApud) [inline Index.html]: Same as 29 line

File: Index.html [hunk 9]
```diff
@@ -1,12 +1,127 @@
...
+        <li>-</li>
+       
+        <li>
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eveniet cum
+        perspiciatis accusantium ratione molestias corrupti. Quo reiciendis
+        quod, totam illum harum omnis culpa expedita, tempora minima deserunt
+        earum eum molestiae, voluptates laboriosam consequatur excepturi
+        provident?
+        </li>
+        <li>-</li>
```
Comment (FernandoRApud) [inline Index.html]: image
Where are this `-`

File: Index.html [hunk 10]
```diff
@@ -1,12 +1,127 @@
...
+      <li class="move-right">designed by Hy-An Multimedia Group</li>
+
+      <li class="move-right">images by unsplash.com</li>
+      </ul>
+      </div>
+
+      
+    </footer>
+  </body>
+</html>
```
Comment (FernandoRApud) [inline Index.html:126]: Apply Prettier please

**Review by FernandoRApud — COMMENTED** [review]
ugu

**Pull request closed without merging**